src/documentation/0000-template/concept.md

```diff
@@ -0,0 +1,83 @@
+---
+title: (What Is a Good Title?)
+description: (~60 character explanation of the article content for page meta data)
+authors: (List of authors/editors, "flaviocopes, jonchurch, your_name_here")
```

Comment:
```suggestion
authors: (List of authors/editors github handles, "flaviocopes, jonchurch, your_name_here")
```
I think it's specifically github handles, with their avatars being pulled from github?